docs/targets/twilio.md

```diff
@@ -40,7 +40,13 @@ indicates that the Bridge was successfully created.
 
 ## Event Types
 
-The Twilio event Target can consume events of any type, however, the Target expects a [JSON][ce-jsonformat] payload at
+A Twilio event Target accepts the following [CloudEvent][ce] types:
+
+
+### io.triggermesh.twilio.sms.send 
+
+Events of this type intend to send a SMS message via Twilio.
+
```

Comment:
This appears disjointed from the additional following line that starts with:
`the [CloudEvent][ce] with the following properties:`